students/DennisLee/lesson02/grid_printer_class.py

```diff
@@ -0,0 +1,78 @@
+"Print a table grid."
+
+
+class TableGrid(object):
+    """
+    Create a square table grid, specifying the number of rows/columns
+    and the cell height/width.
+
+    :rows_and_cols:  The number of rows and columns in the grid.
+
+    :cell_size:  The height and width of a grid cell.
+    """
+
+    def __init(self, rows_and_cols: int, cell_size: int):
```

Comment:
the reason this wasn't working for you is because this is supposed to be `__init__` not `__init`

Reply:
That explains it. And since I used **rows_and_cols** and **cell_size** in the class's methods without prefixing with **self.**, and then used the exact same names in the **if __name__=='main'** clause, those values were in scope to carry over into the methods without causing an error.